src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  box-sizing: border-box;
```

Comment:
move `box-sizing: border-box;` into `*` selector
and remove from other selector